src/main/java/se/sundsvall/chat/integration/db/model/MessageEntity.java

```diff
@@ -0,0 +1,214 @@
+package se.sundsvall.chat.integration.db.model;
+
+import static java.time.OffsetDateTime.now;
+import static java.time.temporal.ChronoUnit.MILLIS;
+
+import jakarta.persistence.CascadeType;
+import jakarta.persistence.CollectionTable;
+import jakarta.persistence.Column;
+import jakarta.persistence.ElementCollection;
+import jakarta.persistence.Entity;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.ForeignKey;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.OneToMany;
+import jakarta.persistence.PrePersist;
+import jakarta.persistence.Table;
+import java.time.OffsetDateTime;
+import java.time.ZoneId;
+import java.util.List;
+import java.util.Objects;
+import org.hibernate.annotations.UuidGenerator;
+
+@Entity
+@Table(name = "message")
+public class MessageEntity {
+
+	@Id
+	@UuidGenerator
+	@Column(name = "id")
+	private String id;
+
+	@Column(name = "sequence_number")
+	private String sequenceNumber;
+
+	@Column(name = "in_reply_to")
+	private String inReplyTo;
```

Comment:
Make it obvious that this is an ID to another message?

`inReplyToMessageId `?